tracpro/orgs_ext/views.py

```diff
@@ -55,7 +55,10 @@ def get_last_contact_sync(self, obj):
         def get_last_flow_run_fetch(self, obj):
             result = obj.get_task_result(constants.TaskType.fetch_runs)
             if result:
-                return format_datetime(ms_to_datetime(result['time']))
+                return "%s (%d fetched)" % (
```

Comment:
Just adds the number of fetched runs to the org view, similar to what there is for contact sync, e.g.

__Last Flow Run Fetch__ Aug 26, 2015 13:16 (0 fetched)